Libplanet.Extensions.Cocona/Commands/KeyCommand.cs

```diff
@@ -4,53 +4,73 @@
 using System.Linq;
 using global::Cocona;
 using Libplanet.Crypto;
-using Libplanet.Extensions.Cocona.Commands.Key;
 using Libplanet.KeyStore;
 
 namespace Libplanet.Extensions.Cocona.Commands
 {
-    [HasSubCommands(
-        typeof(DerivationCommand),
-        "derive",
-        Description = "Derive the address or public key from private key."
-    )]
```

Comment:
If you deprecated the `planet key derive` command and introduce a new command, should `DerivationCommand` be also removed?

Reply:
Wh.... just wow. I just couldn't find the DerivationCommand file from commands folder, so I was like "Well, it says not implemented." Hmm... what do you think? I think in consiceness your implementation is way better, but for simplicity, mine seems better. what's better between yours and mine?

Reply:
I think your implementation seems more simple to use. I think yours is better.